docs/en/use-cases/observability/schema-design.md

```diff
@@ -1112,8 +1112,10 @@ We've effectively reduced the number of rows here from 10m (in `otel_logs`) to 1
 
 Since the merging of rows is asynchronous, there may be more than one row per hour when a user queries. To ensure any outstanding rows are merged at query time, we have two options:
 
-Use the [`FINAL` modifier](/en/sql-reference/statements/select/from#final-modifier) on the table name. We did this for the count query above.
-Aggregate by the ordering key used in our final table i.e. Timestamp and sum the metrics. Typically, this is more efficient and flexible (the table can be used for other things), but the former can be simpler for some queries. We show both below:
+- Use the [`FINAL` modifier](/en/sql-reference/statements/select/from#final-modifier) on the table name (which we did for the count query above).
+- Aggregate by the ordering key used in our final table i.e. Timestamp and sum the metrics. 
+
```

Comment:
Probably a nit, just seemed a little easier to read if we split the second point.